pnpm-workspace.yaml

```
@@ -1,4 +1,3 @@
 packages:
-  - "packages/**"
-
-		
+  - 'packages/exercises'
+  - 'packages/libs/*'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)